README.md

```diff
@@ -161,157 +215,60 @@ with tf.Session() as sess:
+[INSERT]
 
-<img src="https://paper-attachments.dropbox.com/s_194708415DEC35F74A7691FF6810D3B14703D1EFE1672ED29000BA98171242A5_1578695757573_image.png" width="720" alt="Weights & Biases" />
+&nbsp;
 
-Parallel coordinates plots map hyperparameter values to model metrics. They're useful for honing in on combinations of hyperparameters that led to the best model performance.
+# Try out our MLOps Course
 
-<img src="https://i.imgur.com/THYXBN0.png" width="720" alt="Weights & Biases" />
+Get MLOps Certified With The Course From Weights & Biases.
 
-# 📜 Share insights with Reports
-Reports let you [organize visualizations, describe your findings, and share updates with collaborators](https://www.youtube.com/watch?v=o2dOSIDDr1w&&ab_channel=Weights%26Biases).
-
-### Common use cases
-- **Notes:** Add a graph with a quick note to yourself.
-- **Collaboration:** Share findings with your colleagues.
-- **Work log:** Track what you've tried and plan next steps.
-
-**Explore reports in [The Gallery →](https://app.wandb.ai/gallery) | Read the [Docs](https://docs.wandb.com/reports)**
-
-Once you have experiments in W&B, you can visualize and document results in Reports with just a few clicks. Here's a quick [demo video](https://www.youtube.com/watch?v=jWBGKGAjt6w&t=2s&ab_channel=Weights%26Biases).
-
-![](https://i.imgur.com/dn0Dyd8.png)
-
-# 🏺 Version control datasets and models with Artifacts
-Git and GitHub make code version control easy,
-but they're not optimized for tracking the other parts of the ML pipeline:
-datasets, models, and other large binary files.
-
-W&B's Artifacts are.
-With just a few extra lines of code,
-you can start tracking you and your team's outputs,
-all directly linked to run.
-
-### [Try Artifacts in a Colab →](http://wandb.me/artifacts-colab)
-
-![](https://i.imgur.com/zvBWhGx.png)
-
-### Common use cases
-- **Pipeline Management:** Track and visualize the inputs and outputs of your runs as a graph
-- **Don't Repeat Yourself™:** Prevent the duplication of compute effort
-- **Sharing Data in Teams:** Collaborate on models and datasets without all the headaches
-
-![](https://i.imgur.com/w92cYQm.png)
+<p align='center'>
+<a href="https://www.youtube.com/watch?v=ZJtkOX5WcBM&ab_channel=Weights%26Biases"><img src="./README_images/mlops_image.png" /></a>
+</p>
 
-**Learn about Artifacts [here →](https://www.wandb.com/articles/announcing-artifacts) | Read the [Docs](https://docs.wandb.com/artifacts)**
+Bringing machine learning models to production is challenging, with a continuous iterative lifecycle that consists of many complex components. Having a disciplined, flexible and collaborative process - an effective MLOps system - is crucial to enabling velocity and rigor, and building an end-to-end machine learning pipeline that continually delivers production-ready ML models and services.
 
-# 💻  Run W&B Server Locally
```

Comment:
I would also keep this section (maybe in a shortened form) -- many people seems not to know that this it is possible to run server locally.

Reply:
Oops. Missed this. I'll add it in the next PR.